tests/test_job_manager.py

```diff
@@ -8,47 +8,83 @@
 
 """REANA-Job-Controller Job Manager tests."""
 
+import json
 import uuid
 
-from mock import DEFAULT, mock, patch
+import pytest
+from mock import mock
+from pytest_reana.fixtures import sample_workflow_workspace
```

Comment:
Is this necessary? `pytest` fixtures should be automatically loaded when they are set as a test parameter.

Reply:
`[sample_workflow_workspace()](https://github.com/reanahub/pytest-reana/blob/38622303a279fcc39b4a98f22c9b041b42a2eade/pytest_reana/fixtures.py#L445)` is not being loaded, I guess because of `@pytest.fixture(scope='module')`, so I have to import it separately.